MAUI/Themes/Keys.md

```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
     </tr>
 </table>
 
+## SfImageEditor
+
+<table>
+    <tr>
+        <th>Theme Dictionary<br/>
+            <br/></th>        
+        <th>
+          Keys
+            <br/>
+            <br/>
+        </th>
+        <th>
+            Description
+            <br/>
+            <br/>
+        </th>
+    </tr>
+
+    <tr>
+        <td rowspan="19">
+            SfImageEditorStyles  
+            <br/>
+            <br/>
+        </td>
+		<td>
+           SfImageEditorTheme 
+            <br/>
+            <br/>
+        </td>
+        <td>    
+            By merging this key in application resources, it is possible to customize the appearance of the SfImageEditor without merging common theme resource and control style resource dictionaries.
+			
+{% highlight xaml %}
+
+<Application xmlns:syncTheme="clr-namespace:Syncfusion.Maui.Themes;assembly=Syncfusion.Maui.Core"
+             ...>
+ <Application.Resources>
+    <ResourceDictionary>
+        <ResourceDictionary.MergedDictionaries>
+            <syncTheme:SyncfusionThemeResourceDictionary />
+            <ResourceDictionary>
+                <x:String x:Key="SfImageEditorTheme">CommonTheme</x:String>
+                <Color x:Key="SfImageEditorNormalToolbarBackgroundColor">Red</Color>
+                <Color x:Key="SfImageEditorNormalToolbarTextColor">Green</Color>
+            </ResourceDictionary>
+        </ResourceDictionary.MergedDictionaries>
+    </ResourceDictionary>
+ </Application.Resources>
+
+....
+
+</Application>
+
+{% endhighlight %}
+            <br/>
+            <br/>
+        </td>
+	</tr>
+    
+    <tr>
+    <td>
+            SfImageEditorNormalBackground    
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of image Editor control background.
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+    <tr>
+    <td>
+            SfImageEditorNormalToolbarBackgroundColor     
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of toolbar background in image editor.
```

Comment:
Color of toolbar background.

Remove - **in image editor**, or **in Segment** from all the keys